runtime/tests/jvmtitests/agent/agent.c

```diff
@@ -86,7 +86,7 @@ Agent_Prepare(JavaVM * vm, char *phase, char * options, void * reserved)
 		rc = JNI_ERR;
 		goto done;
 	}
-	tprintf(env, 100, "JVMTI version: %p\n", version, getVersionName(env, version));
+	tprintf(env, 100, "JVMTI version: %p (%x) \n", version, version);
```

Comment:
`%p` is not appropriate for a value of type `jint`.

Reply:
Removed `%p`.